apps/flipper_auth/pubspec.yaml

```
@@ -0,0 +1,71 @@
+  flutter_lints: ^5.0.0
+
+# For information on the generic Dart part of this file, see the
+# following page: https://dart.dev/tools/pub/pubspec
+
+# The following section is specific to Flutter packages.
+flutter:
+
+  # The following line ensures that the Material Icons font is
+  # included with your application, so that you can use the icons in
+  # the material Icons class.
+  uses-material-design: true
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 1 but found 2

(indentation)